components/SignInV2.js

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import Container from './Container';
+import { Box, Flex } from './Grid';
+import Image from './Image';
+import Link from './Link';
+import StyledButton from './StyledButton';
+import StyledHr from './StyledHr';
+import StyledInput from './StyledInput';
+import StyledLink from './StyledLink';
+import { Span } from './Text';
+
+const SignInFooterLink = styled(StyledLink)`
```

Comment:
These are internal links, we need to use the `Link` component to prevent a full page reload. I'm pushing a fix for this